sql/GenerateData.php

```diff
@@ -649,7 +649,7 @@ private function addIndividual() {
       }
 
       // Deceased probability based on age
-      if ($age > 40) {
+      if ($age > 60) {
```

Comment:
Thanks @seamuslee001 

Can we try this, if male then 64 years and if female then 68 years.

Refs:
https://www.hcf.com.au/health-agenda/health-care/research-and-insights/life-expectancy-in-australia
https://www.qgso.qld.gov.au/statistics/theme/population/vital-statistics/life-expectancy#current-release-life-expectancy-birth-years-sex-qld-australia